lib/handlers/put.js

```diff
@@ -51,14 +51,16 @@ module.exports.handle = function (buckets, log, protocol, message, done) {
     if (err) {
       agent.metrics.increment('requests.processed.put');
       agent.metrics.histogram('requests.processed.put.time', (new Date() - start));
-      return log.error({
+      var errorContext = {
```

Comment:
it's in extra only for the `errorReporter` call, the log one is still the same. Does this change anything?